pebblo/app/routers/local_ui_routers.py

```diff
@@ -27,3 +27,6 @@
 local_ui_router_instance.router.add_api_route(
     "/not-found/", App.page_not_found, methods=["GET"], response_class=HTMLResponse
 )
+local_ui_router_instance.router.add_api_route(
+    "/app/delete/", App.delete_app, methods=["GET"], response_class=HTMLResponse
```

Comment:
The `DELETE` method would be more appropriate here.

Reply:
Incorporated this. Also added status_code and message in the response, it would be easy for UI to show the result accordingly.